src/components/Utils/orderResource.js

```diff
@@ -31,7 +31,12 @@ export const cloneOrder = (order, mutator, line) => {
 
   delete clonedOrder.id;
   delete clonedOrder.adjustment;
-  clonedOrder.po_lines = [line];
+  if (line) {
+    clonedOrder.po_lines = [line];
+    delete clonedOrder.po_lines[0].purchase_order_id;
```

Comment:
it looks to me as over-engineering. Just move this line above and make it simpler:
`delete line.purchase_order_id;`